tools/ci_build/github/windows/post_code_coverage_to_dashboard.py

```diff
@@ -60,7 +60,7 @@ def write_to_db(coverage_data, args):
     cnx = mysql.connector.connect(
         user='ort@onnxruntimedashboard',
         password=os.environ.get('DASHBOARD_MYSQL_ORT_PASSWORD'),
-        host='onnxruntimedashboard.mysql.database.azure.com',
+        host='172.19.0.7',
```

Comment:
could this be specified in a single place so it's easier to update? similar to the db password

Reply:
I feel search/replace is easy. Adding another indirection would make the code harder to understand. And this is a private ip, we can keep it forever and it won't change.

Reply:
it could also have a more meaningful name than a literal IP address which would be more understandable.

Reply:
Now I move the string to yaml files.

Reply:
that's good. I still see the IP address in a few other places though.

Reply:
I think I can add a yaml file in the template dir like https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/azure-pipelines/templates/set-test-data-variables-step.yml. Then let the other yml files reference it. What do you think?